notebooks/simone_pipeline.md

```diff
@@ -0,0 +1,134 @@
+# Summary of Simone's pipeline
+
+## Overview of methods needed
+
+1. 3d gaussian filter (`skimage.filters.gaussian` or `scipy.ndimage.gaussian`) -> `starfish.filters.gaussian` (already present)
+2. `ndimage.gaussian_laplace` -> `starfish.filters.gaussian_laplace`
+	1. Can we assume that any trimming of the image to fit within the dynamic range should be associated with the function that produces the aberration? E.g. `gaussian_laplace` can produce negative values -- should we wrap that function so that it doesn't have this characteristic?
+  2. If no, we will need some clean-up functions
+3. `max_projection` over z -> `starfish.image._stack` (currently present)
+4. Threshold finding. Is there an equivalent library function for the gradient calculation + trimming that this func does? -> `starfish.stats.calculate_peak_threshold_abs`. Replace `calculate` with your favorite word.
+5. `peak_local_max` -> `starfish.spots.peakfinder` (location from brian-long)
+
+## The pipeline:
+
+- First, `process_standalone_experiment.py` is called. 
+- This is primarily about munging files, and loads in `counting.filtering_and_counting` to do the work.
+
+### Arguments:
+
+1. path - path to experiment
+2. analysis name - prefix for experiment
+3. stringency (default 0) - ? TODO
+4. min-distance (default 5) - minimum distance between peaks
+5. min-plane (default None) - allows selection of a plane to start from (exclude out-of-focus)
+6. max-plane (default None) - allows selection of a plane to end at (exclude out-of-focus)
+
+- Next, `counting.filtering_and_counting` is called
+
+### Arguments:
+
+List Simone's, they're well documented. 
+
+```
+fpath_img_to_filter: str
+path to the file to process
+filtered_png_img_gene_dirs: list
+list of the paths of the directories where the filtered images as are
```

Comment:
interesting -- this reminds me, can you create an issue for the filter CLI and API to write out pngs? also pinging @ttung -- this is something that came up during spec review that users really wanted. we should be able to just learn from simone's code how he handled this (including what he does about negative values)

Reply:
Simone clips negative values. 

#157